api/handlers/metadata.py

```diff
@@ -178,6 +178,34 @@ async def get_meta_thumbnail(
     )
 
 
+def process_geolocation(target: str, geolocation: str):
```

Comment:
Wouldn't be better to receive a geometry here directly instead of a comma separated field?

Reply:
Agree geometry is ultimate flexibility. However, its verbose as a query parameter, it requires more client side code/understanding, and we only have two geolocation fields at present.
The requirements from the customer are for two searches sensor location and transmission location. These are directly addressed at this time in this code as captures and annotations.